### DISCUSSION TITLE
Want to Apply for gsoc 2021 student in the sktime project

### DISCUSSION THREAD

**Discussion by sparkingdark** [alan-turing-institute/sktime#665] [General]
Hello myself Debojyoti,a cs prefinal year student just got to know about the awesome project on time series analysis ,I checked the projects but first how can i get to know where should i start contribution in sktime.

Any mentors help will be helpful.

**Comment by mloning** [comment]
Hi @sparkingdark thanks for reaching out! It's still early in the process, we haven't applied yet and it's not sure that our application will be successful. Having said that we're also collaborating with [INCF](https://incf.org/activities/gsoc) as a mentoring organisation this year. 

For more info, check out the [Ideas page](https://github.com/alan-turing-institute/sktime/wiki/GSoC-2021).

**Reply by sparkingdark** [reply]
Thanks for the advice, now in the meantime I think about to explore te repository and also want to brainstorming on ideas and want to give new ideas as well.

**Reply by mloning** [reply]
I saw that you have some experience with web development. Improving sktime's website may also be a nice project to work on if that's something you're interested in.

**Reply by sparkingdark** [reply]
Yes I have some experience I web development  so yeah I can definitely give
it a try to make sklearn website better in the mean time.

**Reply by sparkingdark** [reply]
Is sktime going to apply under incf as well @mloning

**Reply by mloning** [reply]
What's skyline? Are you free for a call some time soon to discuss this further (please send me a private message on Gitter or via email)?

**Reply by sparkingdark** [reply]
Sure I am available.what is your email address? or I ping you to your twitter. @mloning

**Reply by mloning** [reply]
Yes please DM on Twitter, that also works!

**Reply by sparkingdark** [reply]
follow back so i can dm you.

**Reply by sparkingdark** [reply]
I messaged you.
